Source/options.cpp

```diff
@@ -1316,6 +1319,46 @@ KeymapperOptions::KeymapperOptions()
 	keyIDToKeyName.emplace(SDL_BUTTON_X1 | KeymapperMouseButtonMask, "X1MOUSE");
 	keyIDToKeyName.emplace(SDL_BUTTON_X2 | KeymapperMouseButtonMask, "X2MOUSE");
 
+	keyIDToKeyName.emplace(SDLK_BACKQUOTE, "`");
+	keyIDToKeyName.emplace(SDLK_MINUS, "-");
+	keyIDToKeyName.emplace(SDLK_EQUALS, "=");

```

Comment:
These also have some hardcoded functionality, alongside `SDLK_KP_PLUS` and `SDLK_KP_MINUS`.

https://github.com/diasurgical/devilutionX/blob/09bc43b0edcc05c68d9970a125289e8113d8841b/Source/diablo.cpp#L533-L547